swift_browser_ui/_convenience.py

```diff
@@ -329,6 +329,27 @@ def initiate_os_session(unscoped: str, project: str) -> keystoneauth1.session.Se
     )
 
 
+def os_get_token_from_credentials(
+    username: str,
+    password: str,
+) -> str:
+    """Get an unscoped token with provided credentials."""
+    os_auth = v3.Password(
+        setd["auth_endpoint_url"],
+        username=username,
+        password=password,
+        user_domain_name="Default",
```

Comment:
this needs to be configurabe as i think this openstack installation specific

Reply:
has been made configurable in the lates commit